### ISSUE TITLE
Error during install

### ISSUE THREAD

**Issue by MakaryGo** [YunoHost-Apps/cryptpad_ynh#109]
**How to post a meaningful bug report**
1. *Read this whole template first.*
2. *Determine if you are on the right place:*
   - *If you were performing an action on the app from the webadmin or the CLI (install, update, backup, restore, change_url...), you are on the right place!*
   - *Otherwise, the issue may be due to the app itself. Refer to its documentation or repository for help.*
   - *When in doubt, post here and we will figure it out together.*
3. *Delete the italic comments as you write over them below, and remove this guide.*
--- 

### Describe the bug

I'm unable to install app
### Context

- Hardware: *VPS bought online*
- YunoHost version: 4.2.8.2
- I have access to my server: *Through SSH | through the webadmin*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*
- Using, or trying to install package version/branch: **default**

### Steps to reproduce

- *If you performed a command from the CLI, the command itself is enough. For example:*
    ```sh
    sudo yunohost app install --debug cryptpad
    ```

### Expected behavior

I expected Cryptpad to be installed

### Logs

https://paste.yunohost.org/raw/enesacuxim

**Comment by ericgaspar** [comment]
Hello @MakaryGo, I just reinstalled Cryptpad on my server but I cannot reproduce the bug ...
Maybe you could try installing the new version I just pushed today:
`sudo yunohost app install https://github.com/YunoHost-Apps/cryptpad_ynh/tree/4.11.0`

**Comment by MakaryGo** [comment]
I'll give it a try

**Comment by MakaryGo** [comment]
Same error :/

**Comment by jlnbdt** [comment]
Same error with both main and testing branch :-(

**Comment by Ddataa** [comment]
Hello @MakaryGo and @jlnbdt is this issue still valid ?

**Comment by jlnbdt** [comment]
Nope, not valid anymore. The installation went smoothly for me :)

**Comment by ericgaspar** [comment]
Closing, the package as changed a lot. Please reopen an issue if needed.
